hack/build-and-push.sh

```diff
@@ -90,3 +90,13 @@ docker buildx build \
     --push \
     -f docker/Dockerfile.echo \
```

Comment:
Not necessarily a blocker, but would be nice if you could also rename this image to `echo-advanced` so it was a bit clearer what the difference was.